int64 sign_id
float64 x1_coordinate
float64 y1_coordinate
float64 x2_coordinate
float64 y2_coordinate
---
bool success
float64 x1_intersection
float64 y1_intersection
float64 x2_intersection
float64 y2_intersection